instances/latest/licenses/LGPL-2.0-or-later.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/licenses/LGPL-2.0-or-later",
+  "@type": "https://openminds.ebrains.eu/core/License",
+  "fullName": "GNU Lesser General Public License v2.0 or later",
+  "legalCode": "https://www.gnu.org/licenses/old-licenses/lgpl-2.0-standalone.html",
+  "shortName": "LGPL-2.0-or-later",
+  "webpage": [
+    "https://spdx.org/licenses/LGPL-2.0-or-later.html",
+    "https://spdx.org/licenses/LGPL-2.0+.html"
```

Comment:
I would not refer to the deprecated entry of them

Reply:
I added this entry specifically because someone is referring to this deprecated URL in their codemeta file!

Reply:
okay